src/styles/container.scss

```diff
@@ -0,0 +1,26 @@
+.container {
+  padding: 50px 40px;
+  margin: auto;
+  display: grid;
+  width: 944px;
+  height: 862px;
+
+  grid-template-columns: repeat(1, 1fr);
+  justify-items: center;
+
+  gap: 46px 48px;
+
+  justify-content: space-between;
```

Comment:
The 'justify-content: space-between;' property is not effective in a grid layout as it is intended for flexbox. Consider removing it or using a relevant property for grid layout.